src/blog/2025/08/annual_billing.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Save with FlowFuse Annual Billing – Now Available!"
+subtitle: "Get one month free when you pay annually"
+description: "FlowFuse now offers annual billing for Starter and Pro plans with one month free. Lock in current pricing, simplify administration, and support long-term planning."
+date: 2025-08-22
+authors: ["greg-stoutenburg"]
+keywords: FlowFuse annual billing, Node-RED hosting savings, annual subscription, cost savings, FlowFuse pricing
+tags:
+  - flowfuse
+  - news
+---
+
+We're excited to announce that FlowFuse customers can now choose annual billing for their Starter and Pro plans! Starting today, when you choose to pay for a full year upfront, you'll get one month completely free.
+
+<!--more-->
+
+## How Annual Billing Works
+
+Switching to annual billing is straightforward:
+
+**New customers:** Select annual billing when signing up for Starter or Pro plans
+
+**Existing customers:** Switch to annual billing from your Billing settings

```

Comment:
we could make it a bit more explicit in that users will need to access the upgrade team page from their billing settings and select the same type of team but switch the toggle to yearly, prompting them to yearly billing

<img width="960" height="562" alt="Image" src="https://github.com/user-attachments/assets/f393bc2d-b6aa-4443-aed0-3a9af750d69c" />